Analysis/DataModel/include/Analysis/ReducedInfoTables.h

```diff
@@ -57,14 +58,10 @@ DECLARE_SOA_COLUMN(Pt, pt, float);
 DECLARE_SOA_COLUMN(Eta, eta, float);
 DECLARE_SOA_COLUMN(Phi, phi, float);
 DECLARE_SOA_COLUMN(Charge, charge, short);
-DECLARE_SOA_DYNAMIC_COLUMN(Px, px, [](float pt, float phi) -> float { return abs(pt) * cos(phi); });
-DECLARE_SOA_DYNAMIC_COLUMN(Py, py, [](float pt, float phi) -> float { return abs(pt) * sin(phi); });
-DECLARE_SOA_DYNAMIC_COLUMN(Pz, pz, [](float pt, float eta) -> float { return abs(pt) * sinh(eta); });
-DECLARE_SOA_DYNAMIC_COLUMN(Pmom, pmom, [](float pt, float eta) -> float { return abs(pt) * cosh(eta); });
-
-// MUON tracks extra information
-DECLARE_SOA_COLUMN(MuonChi2, muonChi2, float);
-DECLARE_SOA_COLUMN(MuonChi2MatchTrigger, muonChi2MatchTrigger, float);
+DECLARE_SOA_DYNAMIC_COLUMN(Px, px, [](float pt, float phi) -> float { return pt * cos(phi); });
```

Comment:
Can the existing definition be used?

Reply:
Regarding the definition of pt, eta, phi columns, i would like to have them stored in the skims, not dynamic since most likely we will not skim the signet1Pt, tgl or alpha parameters. But i keep the same naming since this will be useful in templating.
I will update shortly the use of functions from std.